camelot/core/orm/entity.py

```diff
@@ -443,6 +443,16 @@ def get_cls_discriminator(cls):
             if isinstance(discriminator, sql.schema.Column):
                 return getattr(cls, discriminator.key)
             return discriminator
+
+    def is_editable(cls):
```

Comment:
verplaats deze method aub naar de EntityAdmin, zodat we geen extra methods toevoegen op niveau vd Entity.

we willen de 'markering' op niveau vd entity, maar de introspectie zelf, die mag in de Admin zitten

Reply:
done